src/sd_api_v5/sdk/components/serialization/application/codecs/ble/serializers/ble_gap_evt_app.c

```diff
@@ -171,12 +197,15 @@ uint32_t ble_gap_evt_lesc_dhkey_request_dec(uint8_t const * const p_buf,
 
     uint8_t ser_data;
     SER_PULL_uint16(&p_event->evt.gap_evt.conn_handle);
-    
+
     // keyset is an extension of standard event data - used to synchronize keys at application
-    ser_ble_gap_app_keyset_t *keyset;
-    err_code = app_ble_gap_sec_context_find(p_event->evt.gap_evt.conn_handle, &keyset);
+    //TODO: Add proper impl for keyset
+    //ser_ble_gap_app_keyset_t *keyset;
```

Comment:
Looks like there is some unfinished work here.

Reply:
It looks the same as where I changed.
I can change it but I do not have example to test it
which is for the api function ble_gap_evt_lesc_dhkey_request.